src/browser/iframe.js

```diff
@@ -4,54 +4,50 @@
   'use strict';
 
   const Focusable = require('../common/focusable');
-  const {ElementView, BeforeAppendAttribute, VirtualAttribute, Event, VirtualEvent} = require('../common/element');
+  const {Element, BeforeAppendAttribute, VirtualAttribute,
+        BubbledEvent, VirtualEvent} = require('../common/element');
   const React = require('react');
   const {html} = require('reflex');
 
-  class IFrameView extends ElementView {
-    componentDidMount() {
-      super.componentDidMount();
-      this.setState({swapped: true});
+  const DOMProperty = require('react/lib/ReactInjection').DOMProperty;
+
+  // Configure react to make it understand custom mozbrowser attributes.
+  DOMProperty.injectDOMPropertyConfig({
+    Properties: {
+      'remote': DOMProperty.MUST_USE_ATTRIBUTE,
+      'mozbrowser': DOMProperty.MUST_USE_ATTRIBUTE,
+      'mozapp': DOMProperty.MUST_USE_ATTRIBUTE,
+      'mozallowfullscreen': DOMProperty.MUST_USE_ATTRIBUTE,
     }
-    render() {
-      return React.createElement(this.props.type, this.props.model,
-                                 this.props.children);
+  });
+
+  const transplant = (from, to) => {
+    for (let {name, value} of from.attributes) {
+      to.setAttribute(name, value);
+    }
+
+    for (let name of from.properties.names) {
+      to[name] = node[name];
     }
+
+    return to;
   }
 
-  const view = IFrameView.create('iframe', {
+  const view = Element('iframe', {
     opener: new VirtualAttribute((node, current) => {
-      const element = current instanceof String &&
-                      current.unbox &&
-                      current.unbox();
-
-      if (element && node !== element) {
-        for (let {name, value} of node.attributes) {
-          element.setAttribute(name, value);
-        }
+      const element = current instanceof String && current.unbox != null ?
```

Comment:
I should not have, but also played with http://flowtype.org that's why there all this type annotations. Short story is flow can detect those things and in this case type is `() => any|void` so it will not type check if you don't make sure it's not non `nil` :)